docs/contributing/gsoc-ideas.md

```diff
@@ -125,43 +123,57 @@ contributions throughout the application process, including after the
-   your request for
-   help](https://blogs.akamai.com/2013/10/you-must-try-and-then-you-must-ask.html)
-1. you ask your question
-1. someone directs you to a document
-1. you go read that document, and try to use it to answer your question
-1. you find you are confused about a new thing
-1. you ask another question
-1. now that you have demonstrated that you have the ability to read,
-   think, and learn new things, someone has a longer talk with you to
-   answer your new specific question
-1. you and the other person collaborate to improve the document that you
-   read in step 3 :-)
-
-This helps us make a balance between person-to-person discussion and
-documentation that everyone can read, so we save time answering common
-questions but also get everyone the personal help they need. This will
-help you understand the rhythm of help we provide in the developers'
-Zulip livechat -- including why we prefer to give you help in public
-mailing lists and Zulip streams, instead of in one-on-one private
-messages or email. We prefer to hear from you and respond to you in
-public places so more people have a chance to answer the question, and
-to see and benefit from the answer. [More about that in this blog
-post.](https://www.harihareswara.net/sumana/2016/10/12/0)
+## Questions are Important
+
+A successful GSoC revolves around asking well-formed questions.  
+A well-formed question helps you learn, respects the person answering,  
+and reduces the time commitment and frustration level of everyone  
+involved. Asking the right question, to the right person, in the right  
+way, at the right time, is a skill which requires a lifetime of  
+fine-tuning, but Zulip makes this a little bit easier by providing a  
+general structure for asking questions in the Zulip community.
+
+This structure maintain balance between stream discussion and  
+documentation. It saves time answering common questions, while  
+still providing everyone the personal help they need. Becoming familiar  
+and comfortable with this rhythm will be helpful to you as you interact  
+with other developers on [CZO](https://chat.zulip.org). It is always  
+better (and Zulip’s strong preference) to ask questions and have
+conversation through a public stream rather than a private message or  
+an email. This benefits you by giving you faster response times and  
+the benefit of many minds, as well as benefiting the community as others  
+learn from you.
+
+- Stick to the [community norms](https://zulip.com/developer-community/).
+- Read these three blog posts
+  - [Try, then ask](https://www.mattringel.com/2013/09/30/you-must-try-and-then-you-must-ask/)
+  - [We aren’t just making code, we’re making history](https://www.harihareswara.net/sumana/2016/10/12/0)
+  - [How to ask Good Questions](https://jvns.ca/blog/good-questions/)
+- Understand [what makes a great Zulip contributor](..#what-makes-a-great-zulip-contributor)
```

Comment:
I'm pretty sure your test failure is because the leading `..` here is invalid.

You can generally debug by trying to actually click links listed in the CI output.

Reply:
Okay.  I'm getting there.  Now I see how you found the error.  
I'll keep working on it.
Thanks!